### PR TITLE
Ads 598 - Moving ads styling from next apps to next-ads-component

### PR DESCRIPTION
**Pull request by leggsimon**
@robin-marr-ft @andrewgeorgiou1981 @VladDubrovskis

### CODE REVIEW

File: components/section/section.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ export default class Section extends Component {
 								className={sectionAsideClasses}
 								data-trackable={this.props.sidebarComponent.id}>
 								<this.props.sidebarComponent.component articles={this.props.sidebarContent.items} />
-								<div className="sidebar__ad-container"></div>
+								<div className="sidebar-ad-placeholder ad-placeholder"></div>
```
Comment (adgad) [inline components/section/section.js]: is the class sidebar-ad-placeholder used at all?

Comment (leggsimon) [inline components/section/section.js]: @adgad Yes, it's being used here https://github.com/Financial-Times/next-ads-component/blob/a3c85b0bf8cc5ceb1865a41504c31f375ebb1187/src/layouts/front-page/default.js#L28

Comment (adgad) [inline components/section/section.js]: ah cool

**Comment by adgad** [comment]
:+1:

**Pull request merged**